docs/contributor/04-20-lifecycle-management.md

```diff
@@ -0,0 +1,13 @@
+# Lifecycle management of Keda Manager in Kyma
+
+When you enable the Keda module using your Kyma runtime Kyma custom resource (CR), the Lifecycle Manager (LM) downloads the bundled package of the Keda Manager and installs it. Additionally, it applies a sample Keda CR, which triggers Keda Manager to install the Keda module.
+
+![Enable Keda module with LM](/docs/assets/keda-lm-overview.drawio.svg)
```

Comment:
I wouldn't centralize the text in the rectangles with "spec:.."

Reply:
I moved it a little. What do you think?